### PR TITLE
Lpd 19824 - Add support to ERC to Commerce Cart delivery API

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-19824

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/ShippingMethodResourceImpl.java [hunk 1]
```diff
@@ -41,6 +42,44 @@
...
+		if (commerceOrder == null) {
+			throw new NoSuchOrderException(
+				"Unable to find order with external reference code " +
+					externalReferenceCode);
+		}
+
+		CommerceAddress shippingCommerceAddress =
+			commerceOrder.getShippingAddress();
+
+		if (shippingCommerceAddress != null) {
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/ShippingMethodResourceImpl.java:64]: save space

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**